WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -855,62 +840,6 @@ public void onClick(DialogInterface dialog, int id) {
         return false;
     }
 
-    @Override
-    public void openContextMenu(View view) {
```

Comment:
With the context menu removed, we should also drop now-dead calls to `registerForContextMenu()` and `getActivity().openContextMenu()` in all three editor fragments.